ui/main/src/app/modules/admin/components/editmodal/users/edit-user-modal.component.ts

```diff
@@ -126,14 +128,24 @@ export class EditUserModalComponent implements OnInit {
     });
   }
 
+  checkExistingLogin() {
+    if (!! this.userForm.value['login'])
+      this.crudService.getAllUsers().subscribe(users => {
+        if (users.filter(user => user.login === this.userForm.value['login']).length)
+          this.isExistingLogin = true;
+      });
```

Comment:
set this.isExistingLogin = false when no matching user is found

Reply:
Done.